packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -58,7 +58,9 @@ export default class MiddlewarePlugin {
       const location = result
         ? `/${result[1]}`
         : ssrEntryInfo
-        ? entrypoint.name.slice('pages'.length).replace(/\/index$/, '') || '/'
+        ? entrypoint.name
+            .slice(entrypoint.name.startsWith('pages') ? 'pages'.length : 0)
```

Comment:
What's the case that it doesn't start with `pages`?

Reply:
I found `/_error` will go to `or`